recepie_modify.py

```diff
@@ -0,0 +1,39 @@
+#Reference - https://careerkarma.com/blog/python-list-files-in-directory/#:~:text=The%20Python%20os.,in%20an%20entire%20file%20tree.
+import os
+
+path = '/home/vishalraj/AESD/Project/meta-gui/recipes-gui/gui/files'
```

Comment:
@vishalraj3112 the hard coded specific paths should either (ideally) be set by command line arguments or in a single place with comments about customizing